### ISSUE TITLE
Feature Request: MQTT LWT Topic

### ISSUE THREAD

**Issue by pixeldoc2000** [fphammerle/systemctl-mqtt#38]
Please add an MQTT LWT Topic to signal host status like: ON / OFF / DISCONNECTED . It would be great to have the Status of the host of `systemctl-mqtt` im Home-Assistant or any other System.

I know there is `PrepareForShutdown` but i is not the same.

https://www.hivemq.com/blog/mqtt-essentials-part-9-last-will-and-testament/

**Comment by fphammerle** [comment]
Good idea! If anyone wants to start integrating this feature before I can, please go ahead.

**Comment by fphammerle** [comment]
Template for implementation:
https://github.com/fphammerle/switchbot-mqtt/pull/105/files
https://github.com/fphammerle/switchbot-mqtt/commit/31d66b2c66284a85de6c3100888813e294bf10e3
